### PR TITLE
Register jake.is-a.dev

### PR DESCRIPTION
**Pull request by xzrci**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

xzrci.vercel.app

### CODE REVIEW

File: domains/jake.json [hunk 1]
```diff
@@ -1,11 +1,8 @@
 {
-  "description": "Jake's personal website and blog.",
-  "repo": "https://github.com/j-eo/j-eo.github.io",
   "owner": {
```
Comment (cupglassDEV) [inline domains/jake.json:2]: Email key is needed, even tho you dont want it.
Just do this
```suggestion
  "owner": {
  "email":""
  "username:"xzrci"
  }
```

File: domains/jake.json [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
-    "username": "j-eo",
-    "email": "website-contact.nem2h@silomails.com"
+    "username": "xzrci"
   },
```
Comment (cupglassDEV) [inline domains/jake.json:4]: Email key is needed, even tho you dont want it.
Just do this
```suggestion
  "owner": {
  "email": "",
  "username: "xzrci"
  },
```

**Review by cupglassDEV — COMMENTED** [review]
(nevermind, ur good to go)

File: domains/jake.json (hunk 2 above)
Comment (xzrci) [reply domains/jake.json]: i have added, thanks

**Comment by cupglassDEV** [comment]
Oh, no problem.

If you dont want is just add `"email":""`

And before i clicked the review, you already commited a new file 😭

Sorry for that

**Comment by xzrci** [comment]
i edited it after your comment but i really didn't want to share my email. but what can i say

**Comment by xzrci** [comment]
is it okay that I edited an existing jake.json file because it was dead so it would be fine for me

**Comment by cupglassDEV** [comment]
> is it okay that I edited an existing jake.json file because it was dead so it would be fine for me, and further that guy bought a domain but website still doesn't load.

Not ok, talk to the owner first, and the maintainer about that

**Comment by cupglassDEV** [comment]
The example would be like this https://github.com/is-a-dev/register/pull/17033/files, the previous owner and the new owner agrees with the `random.is-a.dev` subdomain takeover

**Comment by cupglassDEV** [comment]
@xzrci

**Comment by xzrci** [comment]
what if owner is nowhere available?

File: domains/jake.json
Comment (Stef-00012) [inline domains/jake.json:1]: This domain is owned by @jake-anto, did they allow you to take it? if so they should comment here to prove it)

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
> i edited it after your comment but i really didn't want to share my email. but what can i say

The email is optional btw, you can remove it (or if you want you can use [OWL](https://owl.is-a.dev), with OWL your email is encrypted and only maintainers can see it

File: domains/jake.json
Comment (iTahseen) [reply domains/jake.json:1]: I can't find him anywhere, even domain isn't pointing to any site. plus he has a site itsjake.me but that even don't load. I don't know how do you deal in such cases if the owner is on long trip to mars.

Comment (xzrci) [reply domains/jake.json:1]: I can't find him anywhere, even domain isn't pointing to any site. plus he has a site itsjake.me but that even don't load. I don't know how do you deal in such cases if the owner is on long trip to mars.

**Comment by xzrci** [comment]
I have updated files. I hope that will work now.

File: domains/jake.json
Comment (Stef-00012) [reply domains/jake.json:1]: On his github profile i see a `fosstodon.org` account, maybe you can try contacting him there?
https://fosstodon.org/@jake_a

Comment (xzrci) [reply domains/jake.json:1]: he posted there 9 months ago ):

Comment (xzrci) [reply domains/jake.json:1]: he posted there 9 months ago ):

Comment (xzrci) [reply domains/jake.json:1]: it's fine if owners don't consider me deserving. they can simply remove this pull.

**Review by Stef-00012 — APPROVED** [review]

**Comment by Stef-00012** [comment]
since the domain isn't reachable and the owner doesn't seem to be reachable, i can merge it

**Comment by xzrci** [comment]
thank you so much ❤️. how long will it take to propagate?

**Comment by jake-anto** [comment]
@xzrci, unfortunately, I'm not on a long trip to Mars :-). And sorry for the late response. I forgot that I set this up, so it's alright if you take over it. Have a nice day!

**Comment by Stef-00012** [comment]
> thank you so much ❤️. how long will it take to propagate?

actually seems like it already did
image

**Comment by Stef-00012** [comment]
if you still see the redirect try clearing your cache
i had this same issue just now, on chrome it kept redirecting and on firefox it worked fine

**Comment by xzrci** [comment]
so finally owner is here, I thought he is having fun there. @jake-anto thank you so much, and brother @Stef-00012 💖

**Comment by xzrci** [comment]
> if you still see the redirect try clearing your cache i had this same issue just now, on chrome it kept redirecting and on firefox it worked fine

thanks, I got one more question, can i somehow use this domain for email I want a mail contact@jake.is-a.dev

**Comment by Stef-00012** [comment]
> > if you still see the redirect try clearing your cache i had this same issue just now, on chrome it kept redirecting and on firefox it worked fine
> 
> thanks, I got one more question, can i somehow use this domain for email I want a mail [contact@jake.is-a.dev](mailto:contact@jake.is-a.dev)

you can, you just need a mail server (some services that offers mails can be https://www.zoho.com or https://improvmx.com), and add it as `MX` record

**Comment by xzrci** [comment]
> > > if you still see the redirect try clearing your cache i had this same issue just now, on chrome it kept redirecting and on firefox it worked fine
> > 
> > 
> > thanks, I got one more question, can i somehow use this domain for email I want a mail [contact@jake.is-a.dev](mailto:contact@jake.is-a.dev)
> 
> you can, you just need a mail server (some services that offers mails are https://www.zoho.com/mail/ or https://improvmx.com), and add it as `MX` record

I have created an account on improvmx.com I hope if it is free, it provided me mx records, do I have to resend another pull?

**Comment by Stef-00012** [comment]
> > > > if you still see the redirect try clearing your cache i had this same issue just now, on chrome it kept redirecting and on firefox it worked fine
> > > 
> > > 
> > > thanks, I got one more question, can i somehow use this domain for email I want a mail [contact@jake.is-a.dev](mailto:contact@jake.is-a.dev)
> > 
> > 
> > you can, you just need a mail server (some services that offers mails are [zoho.com/mail](https://www.zoho.com/mail/) or [improvmx.com](https://improvmx.com)), and add it as `MX` record
> 
> I have created an account on improvmx.com I hope if it is free, it provided me mx records, do I have to resend another pull?

improvmx is free, and yes, you need to make another pull request with the MX record

**Pull request merged**